src/MabeEnum/Enum.php

```diff
@@ -54,7 +54,21 @@
         $this->value   = $value;
         $this->ordinal = $ordinal;
     }
+    
+    /**
+     * Returns a list of all enumeration instances of actual implementation 
+     * 
+     * @return Enum[]
+     */
+    public static function getAll()
```

Comment:
yeah! 

I'll mention here:

- using callable `'self::getByName'`  take execution time  for 8% ca.
- using `foreach` is faster as `array_map` not so wild (-2% - 5% on several versions of PHP and HHVW)
- name of method maybe `listEnums`  or  `getListEnumeration`